src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public String toString(Ball ball) {

```

Comment:
you should override toString() method of Object class where you won't need to pass Ball object